book/src/project-policies.md

```diff
@@ -0,0 +1,61 @@
+# Versioning, Support, and Kubernetes Compatibility
+
+## Versioning
+
+Each Kubernetes CSI component version is expressed as *x.y.z*, where *x* is the
+major version, *y* is the minor version, and *z* is the patch verseion,
+following [Semantic Versioning](https://semver.org/).
+
+Patch version releases only contain bug fixes that do not break any backwards
+compatibility.
+
+Minor version releases may contain new functionality that do not
+break any backwards compatibility (except for alpha features).
+
+Major version releases may contain new functionality or fixes that may break
+backwards compatibility with previous major releases. Changes that require a
+major version increase include: removing or changing API/flags//behavior, new
+RBAC requirements that are not opt-in, new Kubernetes minimum version
+requirements.
```

Comment:
It might be worth explaining how this is tested:
> A litmus test for not breaking compatibility is to replace the image of a component in an existing deployment without changing that deployment in any other way.

Reply:
thanks for the suggestion! added